assignment.js

```diff
@@ -113,6 +131,38 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+let playerAChoice = "scissors"
+let playerBChoice = "paper"
+let outcome = ""
+
+if (playerAChoice == "rock") {
+    if (playerBChoice == "rock") {
+        outcome = "Draw."
+    } else if (playerBChoice == "paper") {
```

Comment:
@justfinish nice touch storing the result of the game in a variable instead of using `console.log()` directly within the conditional. This approach makes your code more flexible.